### PR TITLE
Fix swift & berserker interaction(s)

### PR DESCRIPTION
**Pull request by Ultradragon005**

### CODE REVIEW

**Comment by Ultradragon005** [comment]
bruh I have to think checkdoubletrigger trough

**Comment by Ultradragon005** [comment]
now ready

File: Roles/Core/CustomRoleManager.cs [hunk 1]
```diff
@@ -243,6 +252,15 @@ public static bool OnCheckMurder(ref PlayerControl killer, ref PlayerControl tar
                     DollMaster.CheckMurderAsPossessed(killer, target);
                     return false;
                 }
+
+        if (canceled)
+            return false;
+
+        if (cancelbutkill)
+        {
+            killer.RpcMurderPlayer(target);
```
Comment (Tommy-XL) [inline Roles/Core/CustomRoleManager.cs:261]: But Swift will kill twice and teleport to the dead body

Comment (Ultradragon005) [reply Roles/Core/CustomRoleManager.cs]: oh right, I forgot to change inside swift code 😅 fixed

**Pull request merged**